pkg/resourcemonitor/resourcemonitor.go

```diff
@@ -19,6 +19,7 @@ package resourcemonitor
 import (
 	"context"
 	"fmt"
+	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/k8shelpers"
```

Comment:
since we're moving imports, this belongs in the group below alongside podsfingerprint